src/Test/TestCOModifier.java

```diff
@@ -101,24 +101,24 @@ private boolean testProductionModifier()
     ArrayList<UnitModel> factoryModels = patch.unitProductionByTerrain.get(FC);
     
     // Define a type we want to add, and verify it isn't already buildable.
-    UnitModel bship = patch.getUnitModel(UnitModel.UnitEnum.BATTLESHIP);
-    testPassed &= validate( !factoryModels.contains(bship), "    Factory can build Battleship by default. Malformed test!" );
+    UnitModel ship = patch.unitProductionByTerrain.get(TerrainType.SEAPORT).get(3);
```

Comment:
Ahhh, magic numbers

Reply:
Also this is significantly more unwieldy than just asking for a CO's Battleship model, even if we replace the magic number with an enum or whatever.

Reply:
I do entirely agree, but we can't guarantee there *is* a Battleship for any given unit set.

The magic number is fine IMO since this is test code.